test/extended/extended_test.go

```diff
@@ -11,15 +11,28 @@ import (
 	_ "github.com/openshift/origin/test/extended/jobs"
 	_ "github.com/openshift/origin/test/extended/router"
 	_ "github.com/openshift/origin/test/extended/security"
+	e2e "k8s.io/kubernetes/test/e2e"
+	"os"
+	"testing"
 
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
 // init initialize the extended testing suite.
 func init() {
-	exutil.InitTest()
+	// Kubernetes: Pure end to end tests ~ These flags pass through.
+	if os.Getenv("RUN_KUBE_E2E_SUBSET") == "true" {
+		e2e.RegisterFlags()
```

Comment:
I don't think we want a command exposing completely different flags toggled by envvar... @smarterclayton do you envision the extended test being a binary that honors *exactly* the same flags as upstream e2e? If so, do we even need the envvar guard, and if not, would we be better off with a separate binary?